bokehjs/test/models/glyphs/rect.coffee

```diff
@@ -297,14 +297,14 @@ describe "Rect", ->
           expect(result3['1d'].indices).to.be.deep.equal([1])
 
         it "should work when axis is log", ->
-          data = {x: [60, 100, 140], y: [60, 100, 140]}
+          data = {x: [1, 10, 100, 1000], y: [1, 10, 100, 1000]}
```

Comment:
selenium test seems preferable but let's make a separate issue to add that, and remove this at the same time.